snapm/command.py

```
@@ -2711,6 +2712,38 @@ def main(args):
+            if job == "Other":
+                _log_debug("Look out for the Other!")
+        progress.end("Done!")
+    except KeyboardInterrupt:
+        progress.cancel("Quit!")
+        raise
+
+    try:
+        throbber = ProgressFactory.get_throbber("Doing Something Unbounded", style="bouncingball", no_clear=True)
+        jobs = [f"job{i}" for i in range(0, 500)]
+        throbber.start()
+        for i, job in enumerate(jobs):
```

Ruff (0.14.8)

Loop control variable `job` not used within loop body

(B007)